db/schema.rb

```diff
@@ -62,10 +62,10 @@
     t.integer  "gift_request_id"
     t.integer  "donor_id"
     t.string   "tracking_num"
-    t.boolean  "delinquent"
+    t.boolean  "delinquent",      default: false
     t.string   "state"
-    t.datetime "created_at",      null: false
-    t.datetime "updated_at",      null: false
+    t.datetime "created_at",                      null: false
```

Comment:
I'm sure this is a problem with the way ActiveRecord handles migrations and not anything intentional, but all this whitespace is kind of gross